org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "/app/fireflyluciferin/bin/FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=all",
+    "--filesystem=home",
+    "--env=LD_LIBRARY_PATH=/app/lib",
+    "--env=GTK_THEME=Yaru:dark",
+    "--env=FLATPAK_ENV=1"
+  ],
+  "modules": [
+    "shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json",
+    {
+      "name": "fireflyluciferin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x FireflyLuciferinLinux.deb",
+        "tar xvf data.tar.zst",
+        "mkdir -p /app/fireflyluciferin",
+        "echo -e '\nflatpak.env=true' >> opt/fireflyluciferin/lib/app/classes/project.properties",
+        "cp -r opt/fireflyluciferin/* /app/fireflyluciferin",
+        "install -Dm644 org.dpsoftware.FireflyLuciferin.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+        "install -Dm644 org.dpsoftware.FireflyLuciferin.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png",
+        "install -Dm644 org.dpsoftware.FireflyLuciferin.appdata.xml /app/share/share/appdata/${FLATPAK_ID}.appdata.xml"
+      ],
+      "cleanup": [
+        "/opt"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "org.dpsoftware.FireflyLuciferin.desktop"
+        },
+        {
+          "type": "file",
+          "path": "org.dpsoftware.FireflyLuciferin.appdata.xml"
+        },
+        {
+          "type": "file",
+          "path": "org.dpsoftware.FireflyLuciferin.png"
+        },
+        {
+          "type" : "file",
+          "url" : "https://dpsoftware.org/FireflyLuciferinLinux.deb",
```

Comment:
why not building from source?

Reply:
because at that point of the CI pipeline, I will have the deb file generated, why build again? :)

Reply:
also the URL isn't versioned which mean that the build will break when you override it for a new version.

Reply:
> because it's better.

a CI/CD pipeline will update the checksum when needed, don't worry and in the future I'll switch to a github url.

Reply:
not it won't update the checksum. use a versioned URL. (file or directory name)

Reply:
used a versioned url, thanks